### PR TITLE
Numbers

### PR DESCRIPTION
**Pull request by LiamArmour**
All number tasks have been finished, please can you review. Thanks

### CODE REVIEW

File: numbers/2.convert-celsius-into-farenheit.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+function convertDegreesToFarenheit (){
+    var celsiusValue = 12;
+    var farenheitValue = ((celsiusValue * 2) +30);
```
Comment (keithbarrow) [inline numbers/2.convert-celsius-into-farenheit.js]: Incorrect formula (9/5 c) +32

**Comment by LiamArmour** [comment]
all done @keithbarrow

File: numbers/4.given-the-base-and-perpendicular-lengths-of-a-right-angle-triangle-calculate-the-hypotenuse-length.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+'use strict'
+var hypotenuseLength=0;
+var findhypotenuseLength=function(valueOne,valueTwo){
+    hypotenuseLength=Math.sqrt(((valueOne * valueOne)+(valueTwo * valueTwo)));
```
Comment (keithbarrow) [inline numbers/4.given-the-base-and-perpendicular-lengths-of-a-right-angle-triangle-calculate-the-hypotenuse-length.js:4]: Don't use bracketing where unnecessary:

```
hypotenuseLength=Math.sqrt(valueOne * valueOne + valueTwo * valueTwo);
```

Is easier to read...

**Comment by keithbarrow** [comment]
Single comment - not important, fix & :shipit:

**Pull request merged**